src/text/Heading.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+export interface HeadingProps {
+  /** Heading text */
+  children: React.ReactNode
+  /** Typography style of the heading */
+  typography?: "heading-1" | "heading-2" | "heading-3" | "heading-4" | "subheading" | "strong" | "label" | "small"
```

Comment:
I think "strong" and "smaller" are feeling like odd ones out for me - the rest feel like they offer at least a tiny bit for information on what we should be using the style for, curious for @slowbot thoughts?

Reply:
I know that they were approved before, I just had this thought now that I'm seeing them in context :) I imagine we may need to circle back to decisions we've in the future as well so I hope we can iterate together! I think I wasn't leaning toward the text styles like body or caption but something more related to headings. It still seems odd to me to have this set feel inconsistent, but I am happy to return to this later.

Reply:
@slowbot I'm OK with this change, if it seems like the option of prescribing more semantic names for the heading styles was causing confusion.

Reply:
I like this a lot!

Reply:
Just double-checking the scope of the change here…I'm guessing we still want the `body`, `label` and `caption` semantic size variables, and we're simply renaming the heading 1-4 and subheading variables into `4xl` through `lg`?